portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/VirtualInstancesUpgrade.testcase

```diff
@@ -13,6 +13,27 @@ definition {
 		SearchAdministration.executeReindex();
 	}
 
+	@priority = 4
+	test ViewVirtualInstancesArchive7310 {
+		property data.archive.type = "data-archive-virtual-instances";
+		property database.types = "db2,mariadb,mysql,oracle,postgresql,sqlserver,sybase";
+		property portal.version = "7.3.10";
+
+		ValidateVirtualInstancesUpgrade.viewVirtualInstancesUpgrade();
+	}
+
+	@priority = 4
+	test ViewVirtualInstancesArchive7413U33 {
+		property data.archive.type = "data-archive-virtual-instances";
+		property database.types = "db2,mariadb,mysql,oracle,postgresql,sqlserver";
+		property portal.version = "7.4.13.u33";
+		property test.assert.warning.exceptions = "false";
```

Comment:
I'm initially setting this to false for 7.4.13.u33 because there's an upgrade error relating to Objects. I'll file a bug ticket for Objects and create a follow up ticket to remove this.

Reply:
[LPS-184853](https://issues.liferay.com/browse/LPS-184853)